File: exercise/week01/solution/week01.java
import java.io.IOException;
import java.nio.file.*;
import java.util.List;

public class week01 {
    public static void main(String[] args) {
        try {
            List<String> commands = Files.readAllLines(Paths.get("submarine.txt"));
            int horizontal = 0;
            int depth = 0;

            for (String command : commands) {
                String[] parts = command.trim().split(" ");
                int value = Integer.parseInt(parts[1]);

                switch (parts[0]) {
                    case "forward":
                        horizontal += value;
                        break;
                    case "down":
                        depth += value;
                        break;
                    case "up":
                        depth -= value;
                        break;
                }
            }
            System.out.println("Access Coordinate: " + (horizontal * depth));

        } catch (IOException e) {
            System.out.println("Error: " + e.getMessage() + "\n");
        }
    }
}

File: exercise/week01/solution/week01.php
<?php

// echo "<p>".print_r(file("submarine.txt"))."</p>";

$commands = file("submarine.txt");
// print_r($commands);
// for($i = 0; $i < $commands; $i++) {
//     print_r(explode(" ", $commands[$i]));
//     echo $i;
// }
foreach($commands as $command) {
    print_r(explode(" ", $command));
}